tueplots/fonts.py

```diff
@@ -18,3 +19,33 @@ def icml(*, usetex=False):
         "font.serif": ["Times"],
         "mathtext.fontset": "stix",  # free ptmx replacement
     }
+
+
+def beamer(*, usetex=False):  # for use with the MoML beamer template
+    return {
+        "font.serif": ["Roboto Condensed"],
+        "font.family": "serif",
+        "text.color": colors.tue_dark(),
+        "axes.edgecolor": colors.tue_dark(),
+        "axes.labelcolor": colors.tue_dark(),
+        "xtick.color": colors.tue_dark(),
+        "ytick.color": colors.tue_dark(),
+        "axes.facecolor": "none",
+        "axes.titlesize": "medium",
+        "grid.color": colors.tue_gray(),
+        "axes.grid": False,
```

Comment:
I personally think these are, strictly speaking, not best suited here, but in some `axes` module.
For now there is no better place and I can change this whenever I have a look at this in the future.

Reply:
Yes, this will be something to think about. It might be best to make these a matplotlib [stylesheet](https://matplotlib.org/stable/gallery/style_sheets/style_sheets_reference.html)

Reply:
For the moment, I'm more concerned with how the colors are defined. Is there a reason you want them to be a function? I'd have tried to set them as a static variable

Reply:
see my reply in the issue :)